Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2614 -- Header revision without sign in/out

Resolves [EREGCSC-2614](https://jiraent.cms.gov/browse/EREGCSC-2614)

**Description**

Refactors/revises/rearranges Header component.

**This pull request changes:**

- Removes all code/templates/logic/styles associated with the Flash Banner, the Blocking Modal, and the iframe container components
- Rearranges header to match [Figma design](https://www.figma.com/file/kGg6wtaPK2yxtAayP6JZUb/Authentication-status?node-id=96%3A15675&mode=dev)
   - Styles header components based on responsive break points (mobile/tablet/desktop)
- Updates Cypress tests
- Applies the following style tweaks based on Slack convo with Britta/Jonathan:
   - Crunch down the white gap between the navigation and the page (similar to our current header spacing/styling)
      - Turn the light blue fields at the top of the main area on statute/login/search white (similar to “about” page)
   - 24px instead of 16px between “access statute citations” and “research a subject”
   - Adjust sizing/spacing of “jump to” to match Figma

**Steps to manually verify this change:**

1. Green check marks
2. [Experimental Deployment](https://ummgv9gfmf.execute-api.us-east-1.amazonaws.com/dev1269) loads without side effects
3. Flash banner no longer exists at the top of the page
4. No way to trigger or see the blocking modal that contains the feedback survey
   - the only way to see this survey is via the links near the bottom of the homepage, which open the forms in new, separate tabs
5. Design matches Figma comp
6. All header functionality works as expected
